.eslintignore

```diff
@@ -0,0 +1,4 @@
+lib/*
```

Comment:
How much value would we get from having these other directories linted also? How much effort would it be to do that?